Source/Workers/cesiumWorkerBootstrapper.js

```diff
@@ -63,10 +63,6 @@ var requirejs, require, define;
         return ostring.call(it) === '[object Function]';
     }
 
-    function isArray(it) {
```

Comment:
This is require.js code which should not be modified.

Reply:
added back in ee9854f